### PR TITLE
47 retrieve all gloss properties

### PR DESCRIPTION
**Pull request by leandrumartin**
Small changes just to parse out and store the "themes," "_document," "canonicalReference," "_section," and "_subsection" properties from a Gloss.

### CODE REVIEW

**Comment by leandrumartin** [comment]
Now "description" is also retrieved.

File: src/lib/utils.ts [hunk 1]
```diff
@@ -107,8 +111,12 @@ export function processGloss(gloss: any[], targetId: string): ProcessedGloss {
       processedGloss.targetCollection = item.targetCollection;
     } else if (item.targetChapter && item.targetChapter.value) {
       processedGloss.targetChapter = item.targetChapter.value;
+    } else if (item._section && item._section.value) {
+      processedGloss.targetChapter = item._section.value;
```
Comment (cubap) [inline src/lib/utils.ts:115]: I'll approve this, but I think this is something to deprecate.

Comment (cubap) [reply src/lib/utils.ts:115]: issue #51

File: src/lib/utils.ts [hunk 2]
```diff
@@ -107,8 +111,12 @@ export function processGloss(gloss: any[], targetId: string): ProcessedGloss {
       processedGloss.targetCollection = item.targetCollection;
     } else if (item.targetChapter && item.targetChapter.value) {
       processedGloss.targetChapter = item.targetChapter.value;
+    } else if (item._section && item._section.value) {
```
Comment (cubap) [inline src/lib/utils.ts:114]: This format gives me a few hives. Is there something dangerous in just filtering for a model and object assigning? Not urgent.

**Pull request merged**